src/views/Home.vue

```diff
@@ -3,120 +3,121 @@
 		<div class="flex flex-col min-h-screen bg-primary-gradient">
 			<Navigation />
 			<div
-				class="flex flex-1 h-full items-center container mx-auto h-full"
+				class="flex flex-1 h-full items-center container mx-auto w-full"
 			>
-				<div class="w-2/3 text-white">
+				<div class="mt-4 md:mt-0 px-8 md:px-6 flex flex-col items-center md:items-start w-full md:w-2/3 text-white text-center md:text-left">
```

Comment:
This looks super large to me. I would remove the full height .

![image](https://user-images.githubusercontent.com/27869979/109662326-38d2fa80-7b6b-11eb-924c-9ce8cc7a13b2.png)

Reply:
After I removed h-full nothing changed. I can change size of the text and maybe leading?

Reply:
Sure, it looks terribly large.